### PR TITLE
ARTEMIS-3005 Fix scaling down AMQP messages routing

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownTest.java [hunk 1]
```diff
@@ -379,27 +393,107 @@ public void testScaleDownWithMissingQueue() throws Exception {
    public void testScaleDownWithMissingAnycastQueue() throws Exception {
       final int TEST_SIZE = 2;
       final String addressName = "testAddress";
-      final String queueName1 = "testQueue1";
-      final String queueName2 = "testQueue2";
+      final String queueName = "testQueue";
 
       // create 2 queues on each node mapped to the same address
-      createQueue(0, addressName, queueName2, null, false, null, null, RoutingType.ANYCAST);
+      createQueue(0, addressName, queueName, null, false, null, null, RoutingType.ANYCAST);
```
Comment (michaelandrepearce) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownTest.java:399]: What happened to queue2 this is semantically changing the test case

Comment (michaelandrepearce) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownTest.java:399]: Any fix shouldnt require an existing test to be modified really.

Comment (brusdev) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownTest.java:399]: The queueName1 variable was not used, I have just changed the queue name from `testQueue2` to `testQueue`. My fix doesn't require this change. It just to clean the code and to improve the understandability.

Comment (michaelandrepearce) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/server/ScaleDownTest.java:399]: @brusdev thanks for highlighting that makes sense now +1

**Pull request closed without merging**